src/main/kotlin/no/nav/klage/oppgave/service/distribusjon/KlagebehandlingDistribusjonService.kt

```diff
@@ -42,12 +42,12 @@ class KlagebehandlingDistribusjonService(
                             logger.debug("Vedtak ${vedtak.id} i klagebehandling $klagebehandlingId er ikke distribuert til brevmottaker ${brevMottaker.id}")
 
                             klagebehandling =
-                                lagJournalpostKopierForSekundaereMottakere(klagebehandling, vedtakMedOppdaterteBrevmottakere, brevMottaker)
+                                lagJournalpostKopierForSekundaereMottakere(klagebehandling, vedtak.id, brevMottaker)
```

Comment:
Ser at du byttet ut nesten alle vedtak til vedtak.id. Lurer på om man burde gjort det for `lagBrevmottakere` også, og kanskje da mappe vedtak til vedtakId i `forEach`-en?